paddle/framework/channel_test.cc

```diff
@@ -119,6 +150,7 @@ TEST(Channel, BufferedChannelCloseUnblocksReceiversTest) {
           int data;
           // All reads should return false
           EXPECT_EQ(ch->Receive(&data), false);
+          EXPECT_EQ(data, 0);
```

Comment:
I do not think the `data` should be zero when the channel was closed. If the element of channel is `Tensor*` and the channel has been closed, what will the `data` be after `ch->Receive(&data)`

Reply:
it should be the zero value of that return type, as mentioned here: https://dave.cheney.net/2014/03/19/channel-axioms
Here are the zero values: https://golang.org/ref/spec#The_zero_value

Reply:
The problem here is that C++ syntax doesn't define zero value of all types like Go does. So, basically, we have no idea how to write a C++ code snippet to clear the value of any data type to zero.